dropwizard-jersey/src/main/java/io/dropwizard/jersey/jackson/JacksonMessageBodyProvider.java

```diff
@@ -29,6 +31,7 @@
  * {@link JsonIgnoreType}.)
  */
 public class JacksonMessageBodyProvider extends JacksonJaxbJsonProvider {
+    private static final Logger LOGGER = LoggerFactory.getLogger(JsonProcessingExceptionMapper.class);
```

Comment:
Can you commit the fix and push it back to your `piefel:validationLogging` branch?  Then it should show up in this PR automatically.